tests/unit/customizations/eks/test_update_kubeconfig.py

```diff
@@ -159,6 +159,7 @@ def test_arg_override_first_corrupted(self):
 
 class TestEKSClient(unittest.TestCase):
     def setUp(self):
+        self._proxy_url = "https://myproxy.com"
```

Comment:
it's not clear to me why this (and `self._correct_cluster_entry_with_proxy_url`) needs to be defined here

I'd suggest moving it to inside `test_get_cluster_entry_with_proxy_url_passed`

Reply:
I've moved this into `test_get_cluster_entry_with_proxy_url_passed`